pyproject.toml

```diff
@@ -1,12 +1,23 @@
-[build-system]
-requires = ["build", "setuptools"]
-build-backend = "setuptools.build_meta"
-
-[project]
+[tool.poetry]
 name = "pysquirrel"
 version = "0.1.0"
```

Comment:
I would suggest using the poetry dynamic versioning backed. This poetry plugin automatically sets the version attribute correctly in the pyproject.toml file. 
You can take a look at https://github.com/IAMconsortium/nomenclature/blob/main/pyproject.toml so see how it's used there. From what I remember the two main changes are setting the version number to 0.0.0. That one's not strictly necessary but it communicates that this is just a dummy placeholder. The second change is in the `build-system` section.